addons/website/static/src/scss/primary_variables.scss

```diff
@@ -2068,9 +2068,9 @@ $o-user-website-values: () !default;
 
 // Those are BS values, except BS hardcodes them inside the $hx-font-size
```

Comment:
If we keep these custom sizes here (check my prev. comment), adapt the note removing the "Those are BS values" part